core/src/main/java/org/apache/calcite/plan/RelOptRule.java

```diff
@@ -536,7 +536,7 @@ public RelTrait getOutTrait() {
    * Returns the description of this rule.
    *
    * <p>It must be unique (for rules that are not equal) and must consist of
-   * only the characters A-Z, a-z, 0-9, '_', '.', '(', ')'. It must start with
+   * only the characters A-Z, a-z, 0-9, '-', '_', '.', '(', ')'. It must start with
```

Comment:
This is too small change for a PR, can we just merge it into another bigger one ?

Reply:
> This is too small change for a PR, can we just merge it into another bigger one ?

Yes, I have a pull request for issue(https://issues.apache.org/jira/browse/CALCITE-3136?filter=-1), I can merge this PR to https://github.com/apache/calcite/pull/1275. By the way, @danny0405 do you have time to have a look at https://github.com/apache/calcite/pull/1275. Thanks a lot.

Reply:
@wenhuitang yes merge it with #1275 and close this one ;)